lib/ClangImporter/ImportType.cpp

```diff
@@ -1936,22 +1985,28 @@ Type ClangImporter::Implementation::importMethodType(
   }
 
   clang::QualType resultType = clangDecl->getReturnType();
-  swiftResultTy = importType(resultType, resultKind,
-                             allowNSUIntegerAsIntInResult,
-                             Bridgeability::Full,
-                             OptionalityOfReturn);
+  bool isIUO;
+  std::tie(swiftResultTy, isIUO) =
+      importType(resultType, resultKind, allowNSUIntegerAsIntInResult,
+                 Bridgeability::Full, OptionalityOfReturn);
+
   // Adjust the result type for a throwing function.
   if (swiftResultTy && errorInfo) {
 
     // Get the original unbridged result type.
-    origSwiftResultTy = importType(resultType, resultKind,
-                               allowNSUIntegerAsIntInResult,
-                               Bridgeability::None,
-                               OptionalityOfReturn)
-                            ->getCanonicalType();
-
-    swiftResultTy = adjustResultTypeForThrowingFunction(*errorInfo,
-                                                        swiftResultTy);
+    Type importedType;
+    // FIXME: We drop this on the floor, and should arguably store it
+    // as part of the ForeignErrorConvention created below, but it
+    // doesn't look like clients of that information would necessarily
+    // need it once IUO types are gone from the type system.
```

Comment:
Nah, it doesn't make any difference to us. Throwing functions are throwing functions, regardless of whether the return type was explicitly nullable or unaudited. (Explicitly `null_unspecified` seems very unlikely.)